src/uwtools/logger.py

```diff
@@ -146,8 +147,40 @@ def add_file_handler(cls, logfile_path: Union[str, Path],
         if not logfile_path.parent.is_dir():
             logfile_path.mkdir(parents=True, exist_ok=True)
 
-        handler = logging.FileHandler(str(logfile_path))
+        handler = logging.FileHandler(str(logfile_path), mode="w")
```

Comment:
This is a change in behavior to the log. It will overwrite previous logs, instead of append! Please let me know if this is undesirable.

Reply:
So, I had been looking at this - the log gets kind of unhelpfully cumbersome otherwise. Should we do appending UID or datetime to the log name now or hold it for later with the issue?

Reply:
I think probably we can hold off on that until a little later. We may definitely want that change, though.